src/__tests__/components/fields/range-select/range-select.spec.tsx

```diff
@@ -148,6 +156,17 @@ describe(UI_TYPE, () => {
 		expect(SUBMIT_FN).toBeCalledWith(expect.objectContaining({ [COMPONENT_ID]: { from: "Apple", to: "Cherry" } }));
 	});
 
+	it("should be able to clear all inputs when only 1st option is selected and then click away outside of component", async () => {
+		renderComponent();
+
+		await waitFor(() => fireEvent.click(getComponent()));
+		await waitFor(() => fireEvent.click(getOptionA()));
+		fireEvent.mouseDown(document.body);
+		await waitFor(() => fireEvent.click(getComponent()));
+
+		expect(queryByText(getRangeSelector(), "A")).toBeNull();
```

Comment:
it will be worthwhile to assert the submitted values too

Reply:
done